src/main/java/core/basesyntax/figure/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.figure;
+
+public class IsoscelesTrapezoid extends AbstractFigure implements Figure {
```

Comment:
The class 'IsoscelesTrapezoid' should not extend 'AbstractFigure' if 'AbstractFigure' is being used to set behavior. Consider using an interface to define behavior and an abstract class only for sharing common state or functionality.